rpmlint/checks/DistributionCheck.py

```diff
@@ -1,68 +0,0 @@
-#############################################################################
-# File          : DistributionCheck.py
-# Package       : rpmlint
-# Author        : Frederic Lepied
-# Created on    : Tue Sep 28 00:05:33 1999
-# Purpose       : check the Distribution specificities in a binary rpm package.
-#############################################################################
-
-import re
-
-import rpm
-from rpmlint.checks.AbstractCheck import AbstractCheck
-
-
-class DistributionCheck(AbstractCheck):
-
-    def __init__(self, config, output):
-        super().__init__(config, output)
-        self.man_regex = re.compile(r'/man(?:\d[px]?|n)/')
-        self.info_regex = re.compile(r'(/usr/share|/usr)/info/')
-        self.vendor = self.config.configuration['Vendor']
-        self.distribution = self.config.configuration['Distribution']
```

Comment:
If vendor and distribution config options are no longer used we should kill them from configuration too.

Reply:
they are used, but they're auto-set by our buildsystems (Fedora by Koji, openSUSE by OBS). I'm pretty sure this was a check that was more meaningful in the era of mixed buildsystems and such. Nowadays, there's not really a problem.

Reply:
Ok, removed.